Source/WebCore/layout/formattingContexts/inline/text/TextUtil.cpp

```diff
@@ -460,7 +460,8 @@ bool TextUtil::containsStrongDirectionalityText(StringView text)
     auto length = text.length();
     for (size_t position = 0; position < length;) {
         char32_t character;
-        U16_NEXT(text.characters16(), position, length, character);
+        auto span = text.span16();
+        U16_NEXT(span, position, length, character);
```

Comment:
I assume that `span16()` is a simple enough function, and inlined, so we can just pass `text.span16()` to `U16_NEXT` instead of using a local variable.

Reply:
I'll try again. I seem last time I did that I had some warning about some unused `__c2` variable on the bots and not locally though.

Reply:
Apple has modified our ICU to resolve an unused `__c2` warning, you can see that looking in the Apple copy of the ICU `utf16.h` header.

Reply:
Ok, I'll give it another try. I saw the error on the bots last week though.

Reply:
Maybe bots from non-Apple platforms. This is not important; don’t waste too much time on it please?

Reply:
Still failing on the Mac bots: https://ews-build.webkit.org/#/builders/51/builds/25437